librz/core/analysis_tp.c

```diff
@@ -812,7 +812,7 @@ void propagate_types_among_used_variables(RzCore *core, HtUP *op_cache, RzAnalys
 #define OP_CACHE_LIMIT 8192
 
 RZ_API void rz_core_analysis_type_match(RzCore *core, RzAnalysisFunction *fcn, HtUU *loop_table) {
-	RzListIter *it;
+	void **it;
```

Comment:
Could you move it after the `rz_return...` so that the assert is the first line of the function?